static/src/javascripts/projects/common/modules/ui/sticky.js

```diff
@@ -15,11 +15,11 @@ define([
     var isNewCommercialContent = config.switches.newCommercialContent && config.page.isAdvertisementFeature;
     var paidforBandHeight;
 
-    function initPFBand() {
+    function initPFBand(element) {
```

Comment:
Why not to call it initPaidForBand. Someone who will touch this in half a year will save 2.5 seconds by not wondering what PF means :smile: